README.md

```diff
@@ -490,6 +488,8 @@ For information about the governance of the Node.js project, see
 **David Cai** &lt;davidcai1993@yahoo.com&gt; (he/him)
 * [digitalinfinity](https://github.com/digitalinfinity) -
 **Hitesh Kanwathirtha** &lt;digitalinfinity@gmail.com&gt; (he/him)
+* [lance](https://github.com/lance) -
+**Lance Ball** &lt;lball@redhat.com&gt; (he/him)
```

Comment:
Can you move this so that the list is kept in alphabetical GitHub username order please?

Reply:
> Can you move this so that the list is kept in alphabetical GitHub username order please?

Oh sorry! I was going alphabetical by first name. Will do.